lib/versioneye/crawlers/crates_crawler.rb

```diff
@@ -284,10 +289,10 @@ def self.upsert_version_link(product_db, version_id, name, url)
       language: product_db[:language],
       prod_key: product_db[:prod_key],
       version_id: version_id,
-      link: url
+      name: name
```

Comment:
@timgluz Please revert that to `link`. The `name` of a link can be empty. The `link` itself is mandatory.